modules/backup/main.tf

```diff
@@ -69,4 +69,24 @@ resource "azurerm_storage_management_policy" "graphdb-backup-retention" {
       }
     }
   }
+  # Create a rule to change tiers to cool after 1 day of creation
```

Comment:
I have my reservations whether 1 day is OK, TBH I think this should be at least 3 days. 
Restoring from cold storage is slow, if a backup fails for any reason, the only backup we will have would be in cold storage which would make the restore procedure slow (not sure how slow)

Reply:
Done, it's 3 by default